rfcs/BFDv2/0001-bfd-v2-patient.md

```diff
@@ -0,0 +1,275 @@
+•	http://terminology.hl7.org/CodeSystem/v2-0203  
+
+Patient.gender:
+
+	* Change value set from http://hl7.org/fhir/ValueSet/administrative-gender to http://hl7.org/fhir/ValueSet/administrative-gender|4.0.1
+	
+
+
+#### Not Supported:
+
+•	Telecom field - we don’t send, not in CCW.  
+•	Email field – we don’t send, not in CCW.  
+•	Ethnicity - we don’t send, not in CCW.  
+•	Communication/Language - we don’t send, not in CCW.  
+•	Contact/Next of Kin - we don’t send, not in CCW.  
+
+#### R4 Non-Conformance Errors: 
+
+Next issue ERROR - Patient - Unable to locate profile
+ http://hl7.org/fhir/us/carin/StructureDefinition/carin-bb-patient
+Next issue INFORMATION - Patient.extension[0] - Unknown extension
+
+http://hl7.org/fhir/us/core/StructureDefinition/us-core-race
+
+Next issue INFORMATION - Patient.extension[1] - Unknown extension
+http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity
+
+Next issue INFORMATION - Patient.extension[2] - Unknown extension
+http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex
+
+#### CARIN IG:
+https://build.fhir.org/ig/HL7/carin-bb/StructureDefinition-CARIN-BB-Patient.html
+
+Summary:  
+•	Mandatory: 2 elements  
+•	Must-Support: 1 element  
+
+Slices:
+This structure defines the following Slices:  
+•	The element Patient.identifier is sliced based on the values of value:type, value:system
+
+### Current State
+[Current State]: #current-state
+
+[BFD v1: Current State Patient Resource](./patientReadCurrentState.json)
+
+### Future State
+[Future State]: #future-state
+
+[BFD v2: Future State Patient Resource](./patientReadFutureState.json)
+
+### Differential View
+[Differential View]: #differential-view
+
+
+[Differential_view](./patient-v1-to-v2.diff)
+
+
+### Unresolved Questions
+[Unresolved Questions]: #unresolved-questions
+
+
+The following questions need to be resolved prior to merging this RFC:
+
+1. Hash Identifiers in Patient resource:
+
+Do we need to return hash identifiers? Use the identifier flag in the header to determine what to send – MBI. HICN, Hashes? We do send out BENE_ID as local patient (beneficiary) identifier (medical record number).
+
+**RESOLVED:** Hashed MBI’s will not be provided.
+
+2. Race code mapping for Hispanic:
+
+What is the best approach in CMS-Blue Button to map Hispanic race to US-Core Race code (see below):
+
+	Code			Display  
+•	1002-5			American Indian or Alaska Native  
+•	2028-9			Asian  
+•	2054-5			Black or African American  
+•	2076-8			Native Hawaiian or Other Pacific Islander  
+•	2106-3			White  
+•	UNK			Unknown  
+•	ASKU			Asked but no answer  
+
+**RESOLVED:**  CCW Race codes will be mapped as UNK race code for ALL races to support US-Core race Code System. CCW/SSA Race codes will continue to be provided to accurately reflect race in the system. 		
+
+### Proposed Solution: Drawbacks
+[Proposed Solution: Drawbacks]: #proposed-solution-drawbacks
+
+Supporting both v1 and v2 in parallel may cause slight performance issues.
```

Comment:
What performance concerns do you have here?

Reply:
Once v2 go live and both v1 and v2 traffic are running in parallel, performance and response time will be studied for both v1 and v2 traffic. v2 payloads should hopefully provide some performance gain. v2 will be benchmarked against v1 for performance gain or otherwise.